tests/hikari/impl/test_interaction_server.py

```diff
@@ -61,6 +153,18 @@ def test_when_content_type_provided(self):
         assert response.status_code == 201
 
 
+@pytest.mark.skipif(nacl_present, reason="PyNacl is present")
+def test_interaction_server_init_when_no_pynacl():
+    with pytest.raises(
+        RuntimeError,
+        match=re.escape(
+            "You must install the optional `hikari[server]` dependencies to use the default interaction server."
+        ),
```

Comment:
```suggestion
        match=r"You must install the optional `hikari[server]` dependencies to use the default interaction server."
```
This could also be simplified to this.